user/application_user.md

```diff
@@ -44,7 +48,7 @@ In addition to any environment variables defined here, and later in the job, we
 
 If your application requires secret values, such as S3 secrets, consider leaving them as blank. They can be overrode in a private job.
 
-A volume shared between your services will be mounted at `/share`.
+An ephemeral volume shared only between your services will be mounted at `/share`. Additionally, there is a shared, persistent volume mounted at `/public`. This directory is shared between all application services running on the device, and does not get automatically cleaned up.
```

Comment:
As opposed to the `/share` directory which is not shared between all applications and does get automatically cleaned up? I think a bit more clarify on the difference between these two volumes could be helpful.

Reply:
So `/share` is automatically cleaned up when your job ends, and is only visible to containers within that job? What if "you" schedule multiple jobs onto the same device?